Common/Deployment/DeploymentLib.ps1

```diff
@@ -802,7 +805,7 @@ function InitializeEnvironment()
             UpdateEnvSetting "SubscriptionId" $global:SubscriptionId
         }
     }
-    Select-AzureSubscription -SubscriptionId $global:SubscriptionId
+    
```

Comment:
I believe this is needed as Azure and AzureRM cmdlets can refer to different subscriptions if you don't set both.